usaspending_api/recipient/management/sql/recipient_lookup/000_temp_relations.sql

```diff
@@ -20,10 +20,22 @@ CREATE MATERIALIZED VIEW public.temporary_transaction_recipients_view AS (
     ))::uuid AS recipient_hash,
```

Comment:
UEI is not given priority when generating `recipient_hash` in the above case statement